frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/common/useEditFieldForm.ts

```diff
@@ -65,18 +67,25 @@ export const useEditFieldForm = <FormShape, FieldShape extends FormField>({
   FormShape,
   FieldShape
 >): UseEditFieldFormReturn<FormShape> => {
-  const { stateData, setToInactive, updateEditState, updateCreateState } =
-    useFieldBuilderStore(
-      useCallback(
-        (state) => ({
-          stateData: stateDataSelector(state),
-          setToInactive: setToInactiveSelector(state),
-          updateEditState: updateEditStateSelector(state),
-          updateCreateState: updateCreateStateSelector(state),
-        }),
-        [],
-      ),
-    )
+  const {
+    stateData,
+    setToInactive,
+    updateEditState,
+    updateCreateState,
+    setIsDirty,
+  } = useFieldBuilderStore(
+    useCallback(
+      (state) => ({
+        stateData: stateDataSelector(state),
+        setToInactive: setToInactiveSelector(state),
+        updateEditState: updateEditStateSelector(state),
+        updateCreateState: updateCreateStateSelector(state),
+        isDirty: isDirtySelector(state),
+        setIsDirty: setIsDirtySelector(state),
+      }),
+      [],
+    ),
+  )
```

Comment:
seems like `isDirty` isn't included in the destructuring here, can we take that line out?